Purchases/Public/Error Handling/ErrorUtils.swift

```diff
@@ -241,6 +241,18 @@ import StoreKit
                                 underlyingError: error)
     }
 
+    /**
+     * Constructs an Error with the ``ErrorCode/beginRefundRequestError`` code.
+     *
+     * - Note: This error is used when there is a problem beginning a refund request.
+     */
+    @objc static func beginRefundRequestError(withMessage: String, error: Error? = nil) -> Error {
```

Comment:
In my experience when a param has `with` or `for` we name the variable without it->
 `withMessage message: String`